setup.py

```diff
@@ -52,6 +52,13 @@
     logger.warning("Architecture is not supported by cpuinfo")
     cpuinfo = None
 
+try:  # Embedded copy of cpuinfo
+    from cpuinfo import _parse_arch as cpuinfo_parse_arch
+except Exception:
+    try:  # Installed version of cpuinfo (when installing with pip)
+        from cpuinfo.cpuinfo import _parse_arch as cpuinfo_parse_arch
```

Comment:
For whatever reason when running `pip install .`, the `cpuinfo.py` module at the root is not found.
Since it is installed through the `pyproject.toml` file, let's use it.